app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Notifications MVP
+description: Identifying an initial set of notifications and the critical user flows required to manage notifications in Publish.  
+date: 2020-01-23
+---
+UCAS provided email notifications to it's users. This functionality is yet be introduced to Publish teacher training courses. To address this, we established an introductory set of notifications to satisfy our users critical needs.
+
+We learned through the [accredited bodies research](https://bat-design-history.netlify.com/publish-teacher-training-courses/accredited-bodies-research-round-2#a-need-for-notifications) that there was a need for notifications to be sent to accredited bodies about changes made in Publish. 
+
+## User needs
+
+**As an accredited body**
+I need to be notified of changes by providers in Publish to courses I accredit
+So that I can keep my student record system up to date and in sync with UCAS.
+
+## Critical notifications
+
+Events that would initiate a notification were identified.
+
+1. A course has a basic detail change
+2. A course is created
+3. A course is published
+4. A course is withdrawn
+5. There are no more vacancies in a course (different from withdrawing)
+6. A courses accredited body has changed
+
+The first two templates were to be included in the MVP.
```

Comment:
Not sure what this means

Reply:
Good shout.  

> The first two templates were to be included in the MVP. 

...only makes sense in context of the pilot which is the next entry for Notifications.